grid/recruitment/update-candidate/profile.supr

```diff
@@ -3,7 +3,7 @@ Update Candidate
 Update existing candidate application in ATS (Applicant Tracking System).
 """
 name = "recruitment/update-candidate"
-version = "1.0.0"
+version = "1.1.0"
```

Comment:
suggestion: I miss breezy-hr and workable maps for recruitment/update-candidate@1.1 profile, better open separate PR for this changes

Reply:
My mistake, I did not notice it is already bumped in the maps.